File: 백준/Bronze/2420. 사파리월드/사파리월드.go
package main

import (
	"fmt"
)

func main() {
	var N, M int64
	fmt.Scanf("%d %d", &N, &M)

	if N < M {
		fmt.Println(M - N)
	} else {
		fmt.Println(N - M)
	}
}

File: 백준/Bronze/2420. 사파리월드/사파리월드.c
#define _CRT_SECURE_NO_WARNINGS
#include <stdio.h>
 
int main()
{
    long long N, M;
    scanf("%lld %lld", &N, &M);
 
    if (N < M) {
        printf("%lld", (M - N));
    }
    else {
        printf("%lld", (N - M));
    }
    
    return 0;
}